### PR TITLE
Nicer dates/times for "Detect boot methods"

### PR DESCRIPTION
**Pull request by mk-pmb**

### CODE REVIEW

File: AUTHORS [hunk 1]
```diff
@@ -1,4 +1,6 @@
 Super Grub Disk authors:
 
 Adrian Gibanel Lopez designed and implemented the initial version.
-Jordan Uggla improved Super Grub2 Disk source code.
+Jordan Uggla improved the Super Grub2 Disk source code.
```
Comment (adrian15) [inline AUTHORS]: This should be on a separate commit. So that reverting the functionality change does not revert you being an author.

File: menus/sgd/processoption.cfg [hunk 2]
```diff
@@ -17,14 +17,13 @@
 source "${sg2d_directory}/tools.cfg"
 
 for ncfg in ${sourced_cfgs} ; do
-
+  menu_name="${ncfg}"
```
Comment (adrian15) [inline menus/sgd/processoption.cfg]: You can remove this assignment because this variable is set on the next line.

File: menus/sgd/tools.cfg [hunk 3]
```diff
@@ -172,3 +172,27 @@ function sbasename {
     export $1
 
 }
+
```
Comment (adrian15) [inline menus/sgd/tools.cfg]: Need to test this code to see who it works but overall is ok for me.

**Comment by adrian15** [comment]
And also please make sure that your commit does not change permissions. You are changing some files from 755 to 644 and that's not right.

If they ever need to be 644 there should be a big commit that changes all of the files except for the ones that actually need to be executable.

File: menus/sgd/processoption.cfg (hunk 2 above)
Comment (mk-pmb) [reply menus/sgd/processoption.cfg]: It's the fallback value for the exotic case that the regexp mit not match. I'll clarify it.

File: AUTHORS (hunk 1 above)
Comment (mk-pmb) [reply AUTHORS]: ✔

File: menus/sgd/tools.cfg (hunk 3 above)
Comment (mk-pmb) [reply menus/sgd/tools.cfg]: :+1:
